main.go

```diff
@@ -162,8 +158,24 @@ func main() {
 	output := pwru.NewOutput(&flags, printSkbMap, printStackMap, addr2name)
 	output.PrintHeader()
 
+	outputLimit := flags.OutputLimit
+	if outputLimit == 0 {
+		outputLimit--
```

Comment:
Why this is needed?

Reply:
From code below:
```
for ; outputLimit != 0; outputLimit-- {
```
Negative `outputLimit` decrement will never (unless overflow) reach zero.

Reply:
> Why this is needed?

when `outputLimit` is 0, convert `outputLimit` to 1<<64 - 1. check [https://go.dev/play/p/Cp8Anfq8xBv](https://go.dev/play/p/Cp8Anfq8xBv)

So on, the following `for` loop seems like an infinite loop.


> Negative outputLimit decrement will never (unless overflow) reach zero.

`outputLimit` is an `uint64`, which won't be negative.

Reply:
> loop seems like an infinite loop.

In theory it won't be infinite. `outputLimit = uint64(0); outputLimit--` makes it `max(uint64))`. I'd prefer to avoid such hacks.